packages/scripts/generateBuildConfig.ts

```diff
@@ -3,58 +3,104 @@ import path from "path";
 
 import packageJSON from "../wow-ui/package.json";
 
-type ExportItem =
+const COMPONENT_PATH = "./src/components";
+
+type ExportsKey = string;
+type ExportsValue =
   | {
       types: string;
       require: string;
       import: string;
     }
   | string;
-type ExportObject = { [key: string]: ExportItem };
-type EntryFileObject = { [key: string]: string };
+type ExportsObject = { [key: ExportsKey]: ExportsValue };
 
-const COMPONENT_PATH = "./src/components";
+type EntryFileKey = string;
+type EntryFileValue = string;
+type EntryFileObject = { [key: EntryFileKey]: EntryFileValue };
+
+const getFilteredComponentFiles = async (directoryPath: string) => {
+  const files = await fs.readdir(directoryPath, { recursive: true });
+
+  return files.filter(
+    (file) =>
+      file.endsWith(".tsx") &&
+      !file.includes("test") &&
+      !file.includes("stories")
+  );
+};
 
-const generateExports = (files: string[]) => {
-  const exportsObj: ExportObject = {
-    "./styles.css": "./dist/styles.css",
+const createPaths = (type: "entryFile" | "exports", file: string) => {
+  const componentDirName = file.split("/")[0] ?? "";
+  const componentName = file.split("/")[1]?.slice(0, -4) ?? "";
+  const isComponent = file.endsWith("index.tsx");
```

Comment:
p2;
'/' 로 구분하는게 윈도우 체제에서는 잘 작동하지 않는 것 같아요 ㅠㅠ 
저는 아래처럼 생성된다는,,
![image](https://github.com/GDSC-Hongik/wow-design-system/assets/101736358/f9cabe30-598e-49ad-9a45-159993f51420)

`path` 모듈을 사용해서 경로를 생성하면 윈도우에서도 잘 작동하는 것 같은데 맥에서도 잘 동작한다면 요런 방식으로 수정해보면 어떨까요?
```suggestion
  const parsedPath = path.parse(file);
  const componentDirName = parsedPath.dir.split(path.sep)[0] ?? "";
  const componentName = parsedPath.name;
  const isComponent = componentName === "index";
```

Reply:
아하 이렇게 하니까 맥에서 잘 동작합니다!
변경해뒀습니다~!